doc/topics/releases/fluorine.rst

```diff
@@ -129,3 +129,38 @@ The ``trafficserver`` state had the following changes:
   function instead.
 
 The ``win_update`` state has been removed. Please use the ``win_wua`` state instead.
+
+SaltSSH major updates
+=====================
+
+SaltSSH now can work across different major Python versions. Python 2.7 ~ Python 3.x
+is now supported transparently. Feature works transparently. Requirement is, however,
+that the Master machine should have installed Salt and related dependencies for
+Python 2 and Python 3 should be available and importable there.
+
+SaltSSH also can pack whole additional, another version of Salt. For example, if there
+would be an old box, running only outdated and unsupported Python 2.6, it is still
+would be possible from within Python 3.5 or newer machine to access it. This feature
+requires an additional configuration /etc/salt/master file as follows:
+
+
+.. code-block:: yaml
+
+       ssh_ext_alternatives:
+           2016.3:                     # Namespace, can be actually anything.
+               py-version: [2, 6]      # Constraint to specific interpreter version
+               path: /opt/2016.3/salt  # Main Salt installation
+               dependencies:           # List of dependencies and their installation paths
+                 jinja2: /opt/jinja2
+                 yaml: /opt/yaml
+                 tornado: /opt/tornado
+                 msgpack: /opt/msgpack
+                 certifi: /opt/certifi
+                 singledispatch: /opt/singledispatch.py
+                 singledispatch_helpers: /opt/singledispatch_helpers.py
+                 markupsafe: /opt/markupsafe
+                 backports_abc: /opt/backports_abc.py
+
+It is also possible to add several alternative versions of Salt. One of the way is to
+generate a Minimal tarball using runners and include it. However, this is possible only
+when there is a running alternative Salt version, installed on the Master machine.
```

Comment:
Could this also be rephrased? I'm a little bit confused. But perhaps it's just me. :laughing:

Reply:
Okay, here is my take. Let me know if this still makes sense.

> It is also possible to use several versions of Salt. You can for instance generate a minimal tarball using runners and include that. But this is only possible, when this specific Salt version is also installed on the SaltMaster.

Reply:
Almost. :wink: I took your version and added a bit of things. I think that @terminalmage will come up here at some point and anyways comb that up, since he is very good at it. :stuck_out_tongue_winking_eye: